distribution/lib/Standard/Table/0.0.0-dev/src/Data/Expression.enso

```diff
@@ -4,6 +4,7 @@ polyglot java import java.lang.IllegalArgumentException
 polyglot java import java.lang.UnsupportedOperationException
 polyglot java import org.enso.table.expressions.ExpressionVisitorImpl
 
+## Functions for parsing and handling Enso expressions.
```

Comment:
The only method of this type is `PRIVATE`, shall we make the type `PRIVATE` as well?